includes/asm.h

```diff
@@ -98,6 +102,8 @@ unsigned char					is_not_comment(char *str);
 
 t_file							*read_file(int fd, char *filename);
 
+void							read_binary_champ_code(t_file *file, ssize_t header_size);
```

Comment:
It is more like I had no reason to change the type to unsigned, as it would give no benefits. This variable should be of signed type initially (earlier it is compared with signed value), and I decided not to change the type.

Reply:
Deal. It is not so important here, since we garantie that header size is always positive.